### PR TITLE
Wavefront Integration

### PR DESCRIPTION
**Pull request by oodamien**
**Stream Applications Dashboard (integration-scdf-applications):**
* [x] Streams
* [x] Stream
* [x] Application Runtime
* [x] Application Runtime Instance

**Task Applications Dashboard (integration-scdf-tasks)**
* [x] Tasks
* [x] Task
* [x] Jask execution

Info: https://vmware.wavefront.com/integration/scdf/content

### CODE REVIEW

File: ui/src/app/about/components/about-more/about-details.component.html [hunk 1]
```diff
@@ -50,6 +50,15 @@ <h2>Enabled Features</h2>
             <span class="fa fa-close"></span>
           </strong>
         </div>
+        <div class="line">
+          <span>Wavefront: </span>
+          <strong *ngIf="dataflowVersionInfo.featureInfo.wavefrontEnabled" id="wavefrontEnabled">
```
Comment (ilayaperumalg) [inline ui/src/app/about/components/about-more/about-details.component.html:55]: Does this mean we are going to have a feature flag `wavefrontEnabled` in SCDF (backend)?

Comment (oodamien) [reply ui/src/app/about/components/about-more/about-details.component.html:55]: Yes, see https://github.com/spring-cloud/spring-cloud-dataflow/pull/4093

**Comment by tzolov** [comment]
## Streams
* http://localhost:9393/dashboard/#/streams/definitions
  - Why the per-stream button is enabled while the stream has not been deployed yet?
  - The general button should have the SOURCE set and STREM='*'
  - The per-stream button should open the `integration-scdf-applications` dashboard with SOURCE, and STREAM variables set accordingly while all other variables should be '*'.

* http://localhost:9393/dashboard/#/runtime/apps
  - The per-instance button should set the GUID variable accordingly. 

## Task 
* The http://localhost:9393/dashboard/#/tasks/definitions
  * The general button should sets the SOURCE variable.

* The http://localhost:9393/dashboard/#/tasks/executions
  * The Execution ID variable has to be set.

## General
* Is the `MonitoringDashboardType` being used on the UI side?

**Comment by tzolov** [comment]
Rebased and merged at 135ed19c0d9c5bd1a683558d46f9030c9f6098b8

**Comment by tzolov** [comment]
Rebased and merged at 135ed19c0d9c5bd1a683558d46f9030c9f6098b8

**Pull request closed without merging**